main.py

```diff
@@ -1,7 +1,49 @@
 import init_django_orm  # noqa: F401
 
 from django.db.models import QuerySet
+from db.models import Genre, Actor
 
 
 def main() -> QuerySet:
-    pass
+    genres = ["Western", "Action", "Dramma"]
+
+    actors_name = ["George", "Kianu", "Scarlett",
+                   "Will", "Jaden", "Scarlett"]
+    actors_surname = ["Klooney", "Reaves", "Keegan",
+                      "Smith", "Smith", "Johansson"]
+
+    for genre in genres:
+        Genre.objects.create(
+            name=genre
+        )
+
+    for actors_name, actors_surname in zip(actors_name, actors_surname):
```

Comment:
The loop variables `actors_name` and `actors_surname` are shadowing the lists defined earlier. Consider renaming the loop variables to avoid confusion, such as `first_name` and `last_name`.